test/fixture/assert.snap

```diff
@@ -0,0 +1,19 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
> Can you open an issue on Jest to make it configurable? They seem open to making it more generic when needed.

I don't know, if you look at the [error messages it very much assumes running inside Jest](https://github.com/facebook/jest/blob/38ca942f6f2979ed25f54510173dbd2761dd7e04/packages/jest-snapshot/src/utils.js#L40:L76).

> I'm happy to land this as is for now, but long-term we'll want our own naming and versioning.

I think we want our own alternative, really.